resources/RemoveInactiveUsersRunner.php

```diff
@@ -131,9 +131,16 @@ function sendWarningEmail($user, $elapsedMonths, $deletionThreshold)
             $deletionThreshold . " months.\n\n";
 
     $body .= "Identifiers:\n";
-    foreach ($user->getUserIdentifiers() as $identifier) {
-        $body .= "  - " . $identifier->getKeyName() .": " . $identifier->getKeyValue(). "\n";
-    }
+    
+    $user_ids = $user->getUserIdentifiers();
+    // If a user has identifiers, use them in the email. If not, use the cert DN field.
```

Comment:
What does "use" mean in this context? (See also commit message.)

Reply:
Cool. Did you want to commit that?

Reply:
Done